sympy/core/evalf.py

```diff
@@ -336,9 +336,9 @@ def calc_part(expr, nexpr):
     re, im, re_acc, im_acc = None, None, None, None
 
     if ire:
-        re, re_acc = calc_part(C.re(expr, evaluate=False), ire)
+        re, re_acc = calc_part(sympy.re(expr, evaluate=False), ire)
```

Comment:
```re``` and ```im``` are special, we have local variables named ```re``` and ```im```. I could have renamed them, but not knowing enough of their purpose I couldn't give them a better name, punted on the issue and fell back to ```sympy.re``` and ```sympy.im```.
```sympy.Add``` is one lookup more than ```Add```, so that's the normal pattern I'm using - just a micro-optimization that can't hurt and that I hope will help a bit.